Ledger API: scale all timeLimits in tests with ScaledTimeSpans (#1843)
digital-asset/daml: docs/source/tools/ledger-api-test-tool/index.rst

We used this flag during tool development to ensure that the tool does not
always return success.

Tuning the testing behaviour of the tool
========================================

> Use command line options ``--timeout-scale-factor`` and ``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:

Reviewer: ```suggestion
Use the command line options ``--timeout-scale-factor`` and ``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:
```

Author: Done

> - set ``--timeout-scale-factor`` to floating point value higher than 1.0 to make

Reviewer: ```suggestion
- set ``--timeout-scale-factor`` to a floating point value higher than 1.0 to make
```

Author: Done

  the tool wait longer for expected events coming from the DAML ledger

>   implementation under test, conversely use values smaller than 1.0 to make it

Reviewer: ```suggestion
  implementation under test. Conversely use values smaller than 1.0 to make it
```

Author: Done

  wait shorter;

> - set ``--command-submission-ttl-scale-factor`` to value higher than 1.0 to make

Reviewer: ```suggestion
- set ``--command-submission-ttl-scale-factor`` to a value higher than 1.0 to make
```

Author: Done

>   it generate Ledger API Commands with higher than default Max Record Time,

Reviewer: ```suggestion
  the test tool generate Ledger API Commands with higher than default maximum record time,

Author: Done

>   which might be necessary for DAML ledger implementations which takes long time

Reviewer: ```suggestion
  which might be necessary for DAML ledger implementations which take a long time
```

Author: Done

>   to commit a proposed transaction, conversely use values smaller than 1.0 to

Reviewer: ```suggestion
  to commit a proposed transaction. Conversely use values smaller than 1.0 to
```

Author: Done

We used this flag during tool development to ensure that the tool does not
always return success.

Tuning the testing behaviour of the tool
========================================

Use the command line options ``--timeout-scale-factor`` and
``--command-submission-ttl-scale-factor`` to tune timeouts applied by the tool:

- set ``--timeout-scale-factor`` to a floating point value higher than 1.0 to make
  the tool wait longer for expected events coming from the DAML ledger
  implementation under test. Conversely use values smaller than 1.0 to make it

>   wait shorter;

Reviewer: ```suggestion
  wait shorter.
```

Sorry, missed this the first time around.

Author: I've also updated the capitalization of the first word of the sentences in the bullet points.

Reviewer: Thanks.
